pkg/tfgen/generate_csharp.go

```diff
@@ -1021,6 +1022,10 @@ func (rg *csharpResourceGenerator) generateInputProperty(prop *variable, nested
 		}
 		emitCSharpPropertyDocComment(rg.w, prop, "        ")
 
+		if prop.deprecationMessage() != "" {
+			rg.w.Writefmtln("        [Obsolete(\"%s\")]", prop.deprecationMessage())
```

Comment:
Do we need to do anything to sanitize the string placed in the attribute here? For example, what if the deprecation message happens to have a `"` (quote), then the resulting code won't compile. Or `/` needs to be escaped. etc.

Reply:
You are correct here - I just spoke to @mikhailshilkov about this and we came up with the fix